.github/workflows/if-nodejs-release.yml

```diff
@@ -10,8 +10,8 @@ on:
       - master
       # below lines are not enough to have release supported for these branches
       # make sure configuration of `semantic-release` package mentiones these branches
-      - next
-      - next-major
+      - next-spec
```

Comment:
to get this fixed, please open a PR in `.github` repo -> https://github.com/asyncapi/.github/blob/master/.github/workflows/if-nodejs-release.yml

please also remove `**-release`
also `next-major` is still valid -> for reference https://github.com/asyncapi/parser-js/tree/next-major

Reply:
This PR has been updated.

Reply:
Issued PR agains asycnapi/.github in https://github.com/asyncapi/.github/pull/180

Reply:
But shouldn't be this local workflow be updated as well? Just to reflect reality.

Reply:
once we merge https://github.com/asyncapi/.github/pull/180, the workflow that we use there will make sure to replicate it in every repo that opted in for it, gets it through PR created by bot -> https://github.com/asyncapi/.github/blob/master/.github/workflows/global-replicator.yml#L70-L85

Reply:
right thanks for explaining. Closing this one then.